flink-streaming-java/src/test/java/org/apache/flink/streaming/api/graph/StreamingJobGraphGeneratorTest.java

```diff
@@ -305,4 +308,15 @@ public void invoke(Integer value) throws Exception {
 			}
 		}
 	}
+
+	@Test(expected = IllegalStateException.class)
+	public void testDisabledAutoUIDGenerator() {
```

Comment:
How about adding a test that enabled auto-generated uid

Reply:
That is the current default behavior, it is effectively tested by every other test.